netsuite/Module.md

```diff
@@ -0,0 +1,67 @@
+
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* A NetSuite account
+* Obtaining tokens
+1. Go to [NetSuite](https://www.netsuite.com) and login to your account.
+2. If you have NetSuite Permission to Create Integration APP, The following steps may be helpful or else go to step 03.
+    1. Enable the SuiteTalk Web service features of the account (Setup->Company->Enable Features).
+    
+    2. Create an integration application (Setup->Integration->New), enable TBA code grant and scope, and obtain the 
+    following credentials: 
+        * Client ID
+        * Client Secret
+3. If you have Client ID, Client Secret from your administrator, Obtain the below credentials by following the token based authorization in the [NetSuite documentation](https://www.netsuite.com/portal/developers/resources/suitetalk-documentation.shtml). 
+    * Access Token
+    * Access Token Secret
+4. Obtain the SuiteTalk Base URL, which contains the account ID under the company URLs (Setup->Company->Company
+    Information).
+
+        E.g:  https://<ACCOUNT_ID>.suitetalk.api.netsuite.com
+
+## Quickstart
+To use the NetSuite connector in your Ballerina application, update the .bal file as follows:
+### Step 1 - Import connector
+* Create a Ballerina file and import the following Netsuite module and the others. 
+```ballerina
+import ballerinax/netsuite;
+```
+### Step 2 - Create a new connector instance
+2. Create a NetSuite Client by providing credentials. Initialize the connector by giving authentication details in the HTTP client config, which has built-in support for Token Based Authentication(TBA). NetSuite uses TBA to authenticate and authorize requests. The NetSuite connector can be initialized by the HTTP client config using the client ID, client secret, access token and access token secret. You may use Ballerina [Configuration variable](https://ballerina.io/learn/by-example/configurable.html) to store your credentials.
+```ballerina
+public function main() returns error? {
+    netsuite:NetsuiteConfiguration nsConfig = {
+        accountId: "<accountId>",
+        consumerId: "<consumerId>",
+        consumerSecret: "<consumerSecret>",
+        token: "<token>",
+        tokenSecret: "<tokenSecret>",
+        baseURL: "<webServiceURL>"
+    };
+
+    netsuite:Client netsuiteClient = check new(nsConfig);
+```
+### Step 3 - Invoke connector operation
+1. Invoke connector operation using the client
+```ballerina
+    netsuite:RecordInputRef currency = {
+        internalId: "1",
+        'type: "currency"
+    };
+    netsuite:NewAccount account = {
+        acctNumber: "1234567",
+        acctName: "Ballerina NetSuite Demo Account",
+        currency: currency
+    };
+    netsuite:RecordAddResponse newAccount = check netsuiteClient->addNewAccount(account);
+}
+```
+2. Use `bal run` command to compile and run the Ballerina program.
```

Comment:
Is this a part of "Step 3 - Invoke connector operation"? Can't this be step 4?

Reply:
This goes with the agreed format in all connectors, I'll check